sdccc/src/main/java/com/draeger/medical/sdccc/messages/mapping/ManipulationData_.java

```diff
@@ -25,6 +25,7 @@ public final class ManipulationData_ {
     public static volatile SingularAttribute<ManipulationData, Long> startTimestamp;
     public static volatile SingularAttribute<ManipulationData, Long> finishTimestamp;
     public static volatile SingularAttribute<ManipulationData, ResponseTypes.Result> result;
+    public static volatile SingularAttribute<ManipulationData, Byte[]> response;
```

Comment:
Probably should be

```suggestion
    public static volatile SingularAttribute<ManipulationData, String> response;
```

Why does it work this way?